prereq_map/context_processors.py

```diff
@@ -0,0 +1,10 @@
+from django.conf import settings
+
+
+def google_analytics(request):
+
+    ga_key = getattr(settings, 'GOOGLE_ANALYTICS_KEY', False)
+    return {
+        'GOOGLE_ANALYTICS_KEY': ga_key,
```

Comment:
Why does this one get returned as well? Don't we just use {{ google_analytics }} in the template?